scripts/pcmping

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python
+
+"""
+    Portchannel member ping: get portchannel member packet loss state.
+
+"""
+
+from scapy.all import *
+import argparse
+import random
+import socket
+import select
+import time
+import ipaddr as ipaddress
+import sys
+import swsssdk
+
+RCV_SIZE_DEFAULT = 4096
+RCV_TIMEOUT = 10000
+DEFAULT_PACKET_LEN = 100
+SELECT_TIMEOUT = 2
+
+def receive(my_socket, timeout, exp_pkt, exp_socket, time_sent):
+    timeLeft = timeout
+    delay = 0
+    while True:
+        startedSelect = time.time()
+        whatReady = select.select(my_socket, [], [], timeLeft)
+        if whatReady[0] == []: # Timeout
+            return False, delay
+        timeReceived = time.time()
+        
+        for sel in whatReady[0]:
+            recPacket = sel.recv(4096)
+            e = str(exp_pkt)
+            # exclude ethernet header
+            p = str(recPacket[14:])
+            if e == p:
+                if sel == exp_socket:
+                  delay = (timeReceived - time_sent) * 1000
+                  return True, delay
+        timeLeft = timeLeft - (timeReceived - startedSelect)
+        if timeLeft <= 0:
+            return False, delay
+        
+def create_decap_packet(interface, dst_out, dst_in, sockets, exp_socket, max_iter):
+    dscp = 16
+    tos = dscp << 2
+    tcp_hdr = TCP(sport=1234, dport=80, flags="S", chksum=0)
```

Comment:
>80 [](start = 36, length = 2)

the return packet will be a tcp port to port 80, it can confuse the application if sonic opens port 80.

can you create icmp reply packet so that it won't harm.